kyuubi-server/src/main/scala/org/apache/kyuubi/engine/JpsApplicationOperation.scala

```diff
@@ -68,7 +68,11 @@ class JpsApplicationOperation extends ApplicationOperation {
         (true, s"Succeeded to terminate: $idAndCmd")
       } catch {
         case e: Exception =>
-          (false, s"Failed to terminate: $idAndCmd, due to ${e.getMessage}")
+          if (getEngine(tag).nonEmpty) {
+            killApplicationByTag(tag)
+          } else {
+            (false, s"Failed to terminate: $idAndCmd, due to ${e.getMessage}")
+          }
```

Comment:
what is this for, seems not related to the scope of pr

Reply:
yeah, we can fix this in a new pr

Reply:
so, what are we going to fix?

Reply:
the spark-submit may generate two process. 

And we might fail to kill the running batch jobs after each test in BatchResourceSuite.

Reply:
It might cause that the new added UT failed.

Because the new UI assert that the running count is 0 at the begining.

Reply:
let's not hide bugs...

Reply:
have remove it from this pr.